File: w09/d01/edward_shin/mail_it/spec/mail_item_spec.rb
require_relative 'spec_helper'
require_relative '../mail_item'
require_relative '../package'
require_relative '../postcard'

describe Mail_Item do

  it "should have a ship_to_address" do
    mail_item = Mail_Item.new
    expect(mail_item.ship_to_address).to be_true
  end

  it "should have a stamp value" do
    mail_item = Mail_Item.new
    expect(mail_item.stamp_value).to be_true
  end

end

File: w09/d03/matthew_korporaal/mail-object/mail_object_app/spec/javascripts/mailitem_spec.js
describe(".mailItem", function () {

  describe("Properties", function () {

    beforeEach(function () {
      mailitem = new MailItem();
    });

    it("should have a shipToAddress string", function () {
      expect(mailitem.shipToAddress).toBeDefined();
    });

    it("should have a stampValue number", function () {
      expect(mailitem.stampValue).toBeDefined();
    });
  });
});